src/Components/ProvisioningWizard/steps/Pubkeys/Pubkeys.test.js

```diff
@@ -19,6 +19,33 @@ describe('Pubkeys', () => {
       expect(items).toHaveLength(2);
     });
 
+    test('view more with click', async () => {
```

Comment:
Wow, this seems to work, because the tests are green but I wonder ... HOW? xD
We have the limit set to 100 constant and we mock only the first request...
Do you know why it actually works? :open_mouth:

Reply:
The test is green because, for each launch click, a new pubkey is created, and in the test teardown, the pubkey is removed.....so in a way, dropdown of pubkey has only 1 pubkey present. 
The test will fail when it comes to integration test since there might be pubkeys present of other users.